feature-extensions/azure/jenkins_config.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+class colors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+def update_jenkins(jenkins_user, jenkins_host, jenkins_port, jenkins_api_token, azure_client_id, azure_client_secret,
+                   azure_tenant_id, azure_subscription_id):
+    basic_auth = HTTPBasicAuth(jenkins_user, jenkins_api_token)
+
+    add_jenkins_credential(jenkins_host, jenkins_port, basic_auth, 'AZ_CLIENTID', azure_client_id)
+    add_jenkins_credential(jenkins_host, jenkins_port, basic_auth, 'AZ_PASSWORD', azure_client_secret)
+    add_jenkins_credential(jenkins_host, jenkins_port, basic_auth, 'AZ_SUBSCRIPTIONID', azure_subscription_id)
+    add_jenkins_credential(jenkins_host, jenkins_port, basic_auth, 'AZ_TENANTID', azure_tenant_id)
+
+
+def add_jenkins_credential(jenkins_host, jenkins_port, basic_auth, key, value):
+    url = "http://{}:{}/credentials/store/system/domain/_/createCredentials".format(jenkins_host,
+                                                                                    jenkins_port)
+    content = """{{
+  "": "0",
```

Comment:
Fix formatting.

Reply:
Replaced with non-multiline string and formatted.